draft/draft-scharf-tcpm-yang-tcp.xml

```diff
@@ -101,14 +101,35 @@
       means to configure the parameters listed in this document, which are
       outside the scope of this document.</t>
 
-      <t>This specification is orthogonal to <xref target="RFC4022">Management
+      <t>This specification does not deprecate the <xref target="RFC4022">Management
       Information Base (MIB) for the Transmission Control Protocol
-      (TCP)</xref>. The <xref target="RFC4898">TCP Extended Statistics
-      MIB</xref> is also available. It is possible to translate a MIB into a
+      (TCP)</xref>. The basic statistics defined in this document follow
+      the model of the TCP MIN. An <xref target="RFC4898">TCP Extended Statistics
```

Comment:
I do not know what "TCP MIN" means. If you think the community knows and understands what the term means, then you do not have to expand on it. Otherwise, I would add some description around it.